pycloudflare/utils.py

```diff
@@ -0,0 +1,32 @@
+from functools import wraps
+
+from pycloudflare.exceptions import SSLUnavailable
+from pycloudflare.services import HTTPServiceError
+
+
+errors_mapping = {
+    1001: SSLUnavailable,
+}
```

Comment:
I raised [a ticket](https://support.cloudflare.com/hc/en-us/requests/1363639) with CF support, you're CCed.